ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -743,15 +743,15 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
             return
         }
 
-        if dimmingView.alpha == 0 {
-            dimmingView.isAccessibilityElement = false
-            view.accessibilityViewIsModal = false
-        } else {
+        if dimmingView.alpha == 1 {
```

Comment:
With the change to make alpha explicitly 1 or 0, is this part needed? In fact, 0 may technically be the more correct value since it appears we're checking alpha as an indirect proxy for whether the view is hidden. From that lens, hidden=false usually means alpha between 0.00001 and 1 and hidden=true usually means alpha == 0.

Natural follow-up question - should we be explicitly hiding this view when it's at 0 alpha and can we then key off the `hidden` property instead?

Reply:
I think I agree with this. Lots of potential for simplification here I think, because if we actually set `hidden` to the correct value on completed state change, we could get away with keeping `isAccessibilityElement` set to true, right? A hidden view should not show up in a11y hierarchy automatically.

I'd also recommend revisiting the need for `accessibilityFrame` updates - it seems a little sus we need to do this. Overlapping a11y elements should naturally appear across the system, so I don't understand why we need to handle that case here specifically.

Btw I realize this is scope creep but I see potential for tons of code deletion here :D

Reply:
this is so so so close to working with one caveat: we lose the alpha animation when tapping on the resizing handle and it snaps from hidden to not hidden. since changing from 0 alpha to hidden is not a 1 to 1 swap, i'm leaving as is for now to get this fix to customers before the end of the month